README.md

```diff
@@ -81,7 +91,7 @@ body that says "hello".
 
 Your Answer:
 ```js
-// your code here
+$("#greeting").on("click", this.append("<p>hello</p>"));
```

Comment:
The question asks you to append the "p" element to the "body", not to the element that was clicked ("this").